.github/workflows/pr.yml

```diff
@@ -19,26 +15,26 @@ env:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
-    steps:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+         - ubuntu-latest
+         - macos-latest
+         - windows-latest
+    steps:          
       - uses: actions/checkout@v4
 
       - name: Bootstrap Action Workspace
         id: bootstrap
         uses: ./.github/actions/bootstrap
 
       - name: Build
-        run: ./build.sh
+        run: dotnet run --project build -c release
 
       - name: Test
-        run: ./build.sh test
+        run: dotnet run --project build -c release -- test
 
       - name: Publish AOT
-        run: ./build.sh publishbinaries
-        
-      - uses: actions/upload-artifact@v4
```

Comment:
I wonder what we were using this for?

Reply:
This was there in the initial implementation of the preview build, where we would download the build binary.